source/_patterns/01-molecules/15-loan-sections/18-loan-story.mustache

```diff
@@ -67,7 +67,7 @@
 					<img class="reviewer-image" src="{{reviewerImageLink}}" alt="Translator profile picture"/>
 				{{/reviewerImageLink}}
 				<p>
-					Translated from {{language}} by {{{reviewerLink}}} {{reviewerName}}
+					Translated from {{language}} {{#reviewerLink}} by {{{reviewerLink}}} {{reviewerName}}{{/reviewerLink}}
```

Comment:
@coderintherye - I think I should have looked at this more closely last time. Look at the getTranslatorByline() function on line 762 of Volunteer.php, where API::volunteer()->getTranslatorByline($field_partner_loan) gets its data. It looks like it will return a  reviewer_link for any non-English loan. But it only sets a reviewer_name if ($reviewer->showName && $reviewer->bylineName). So, looking at the ticket, I think the condition should actually be reviewerName and not reviewerLink. If I'm wrong then this is fine, else maybe one more tweak.